pkg/auth/handler/webapp/send_oob_otp.go

```diff
@@ -0,0 +1,127 @@
+package webapp
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/authgear/authgear-server/pkg/auth/handler/webapp/viewmodels"
+	"github.com/authgear/authgear-server/pkg/auth/webapp"
+	"github.com/authgear/authgear-server/pkg/lib/authn"
+	"github.com/authgear/authgear-server/pkg/lib/infra/mail"
+	"github.com/authgear/authgear-server/pkg/lib/interaction"
+	"github.com/authgear/authgear-server/pkg/util/httproute"
+	"github.com/authgear/authgear-server/pkg/util/phone"
+	"github.com/authgear/authgear-server/pkg/util/template"
+)
+
+const AuthenticatorIndex = 0
```

Comment:
Since we sort the default authenticators as the first item by convention, I think we can just inline this constant, or name it `DefaultAuthenticatorIndex`.